datalad/core/local/save.py

```diff
@@ -130,14 +130,15 @@ class Save(Interface):
             args=("--to-git",),
             action='store_true',
             doc="""flag whether to add data directly to Git, instead of
-            tracking data identity only.  Usually this is not desired,
-            as it inflates dataset sizes and impacts flexibility of data
-            transport. If not specified - it will be up to git-annex to
-            decide, possibly on .gitattributes options. Use this flag
-            with a simultaneous selection of paths to save. In general,
-            it is better to pre-configure a dataset to track particular paths,
+            tracking data identity only.  Use with caution, there is no
+            guarantee that a file put directly into Git like this will
+            not be annexed in a subsequent save operation.
```

Comment:
I personally find this unclear.  It sounds like it could be expected behavior `save()` for a save operation to throw _unmodified_ git files into annex.

Guess you didn't like my suggestion in gh-1974, but I think that avoids the issue.

Reply:
I think given the behavior shown in gh-1974 we have to discourage this option as much as possible. I did not consider the possibility to interpret my changes in the way you point out. But just the change you suggested did not emphasize the potential pitfalls enough for my taste. Personally, I think if the new wording discourages this kind of use more than necessary, it is still a good thing.

Ultimately, I think we have to make such a call add matching `.gitattribute` settings.

Reply:
> Personally, I think if the new wording discourages this kind of use more than necessary, it is still a good thing.

I don't think wording things in a way that could lead to a misunderstanding is a good thing, even if it leads to a particular behavior that we see as desirable.  Anyway, we don't seem to agree on this, it doesn't particularly matter, and it is already merged, so I'll drop it.